plugins/platform-portal-backstage-plugin-backend/src/provider/GlooPlatformPortalProvider.ts

```diff
@@ -529,4 +543,12 @@ export class GlooPlatformPortalProvider implements EntityProvider {
 
     return processedAPIs;
   }
+
+  private async applyEntityMapping(entity: Entity, api: ApiVersionExtended | API) {
+    if (this.entityTransformation) {
```

Comment:
In our UI code, we try to be explicit when checking variable truthiness by converting it to a boolean first, like this (`!!value` is the same as `Boolean(value)`).

```suggestion
    if (!!this.entityTransformation) {
```

Reply:
done , thanks a lot